frontend/src/i18n/locales/en-US.json

```diff
@@ -68,6 +68,7 @@
     "question-circle-warning": "Don't use this feature unless you know what you're doing.",
     "learn-more": "Learn more",
     "select-token": "Select token to send",
+    "slippage-exceeded": "Actual send amount would materially differ from the estimate due to changing gas prices. Please try again.",
```

Comment:
Coming back to this, I still think "materially" is what we want and "significantly" is incorrect/confusing. We throw if the final amount is more than 2% off of the projected amount. For a $10 transaction, that's an error of $0.20, an insignificant amount, but it might still matter to someone (i.e. be "material")